src/input-capture.c

```diff
@@ -31,7 +31,7 @@
 #include "xdp-impl-dbus.h"
 #include "xdp-utils.h"
 
-#define VERSION_1  1 /* Makes grep easier */
+#define VERSION  2 /* Makes grep easier */
```

Comment:
```suggestion
#define VERSION_1 1 /* Makes grep easier */
#define VERSION_2 2
```
And then use `VERSION_2` where we want that newer version.